src/app/users/create-user/create-user.component.html

```diff
@@ -0,0 +1,127 @@
+<div class="container">
+
+  <mat-card>
+
+    <form [formGroup]="userForm" (ngSubmit)="submit()">
+
+      <mat-card-content>
+
+        <div fxLayout="row wrap" fxLayoutGap="2%" fxLayout.lt-md="column">
+
+          <mat-form-field fxFlex="48%">
+            <mat-label>Username</mat-label>
+            <input matInput required formControlName="username">
+            <mat-error *ngIf="userForm.controls.username.hasError('required')">
+              Username is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field fxFlex="48%">
+            <mat-label>Email</mat-label>
+            <input matInput *ngIf="userForm.controls.sendPasswordToEmail.value === true" required formControlName="email">
+            <input matInput *ngIf="userForm.controls.sendPasswordToEmail.value === false" formControlName="email">
```

Comment:
A better solution to this would be:
```
            <input matInput [required]="userForm.controls.sendPasswordToEmail.value" formControlName="email">
```